samples/azure-quantum/hello-world/HW-rigetti-qsharp.ipynb

```diff
@@ -0,0 +1,347 @@
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "0cc00d71-e8d8-447a-b40f-5cc1784002e0",
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "import qsharp.azure\n",
+        "targets = qsharp.azure.connect(\n",
+        "   resourceId=\"/subscriptions/916dfd6d-030c-4bd9-b579-7bb6d1926e97/resourceGroups/AzureQuantum/providers/Microsoft.Quantum/Workspaces/alchocro-workspace-for-demo\",\n",
+        "   location=\"westus2\")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "dcd6f6c2-98f5-4a85-900e-6d06625baca7",
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "source": [
+        "We can use the resulting object to see which _targets_ are available for submission."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "013e6d23-d364-4926-93b9-4b829a9d85d7",
+      "metadata": {
+        "jupyter": {
+          "outputs_hidden": false,
+          "source_hidden": false
+        },
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "outputs": [],
+      "source": [
+        "import qsharp\n",
+        "\n",
+        "print(\"This workspace's targets:\")\n",
+        "for target in targets:\n",
+        "    print(\"-\", target.id)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "06babf11-5463-4890-a9da-14aa0add09e6",
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "source": [
+        "### ❕ Do you see `rigetti.sim.qvm` in your list of targets? If so, you're ready to keep going.\n",
+        "\n",
+        "Don't see it? You may need to add Rigetti to your workspace to run this sample. Navigate to the **Providers** page in the portal and click **+Add** to add the Rigetti provider."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "9c74a673-6d07-4dab-8c28-c0b47e75365b",
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "source": [
+        "## Rigetti: The quantum provider\n",
+        "Azure Quantum partners with third-party companies to deliver solutions to quantum jobs. These company offerings are called *providers*. Each provider can offer multiple *targets* with different capabilities. See the table below for Rigetti's targets.\n",
+        "\n",
+        "| Target name | Target ID | Number of qubits | Description |\n",
+        "| --- | --- | --- | --- |\n",
+        "| Rigetti QVM (simulator) | `rigetti.sim.qvm` | ? qubits | Rigetti's cloud-based \"Quantum Virtual Machine\" simulator. Can run with or without noise. |\n",
```

Comment:
Can the QVM be run with noise? I thought that wasn't currently possible? If so, we should make sure to include in docs how to configure noise models.

Reply:
Yes, when used in an offline mode, but I thought that wasn't currently exposed?

Reply:
If that's the case, I'll remove the mention of it